app/src/components/map/filters_panel.jsx

```diff
@@ -1,18 +1,24 @@
 import React, { Component } from 'react';
 import filtersPanel from '../../../styles/components/c_filter_panel.scss';
 import { FLAGS } from '../../constants';
+import iso3311a2 from 'iso-3166-1-alpha-2';
 
 class FilterPanel extends Component {
 
   render() {
     let countries = Object.keys(FLAGS).map((index) =>
-      <option value={index} key={index}>{FLAGS[index]}</option>
+      <option
+        value={index}
+        key={index}
+      >
+        {iso3311a2.getCountry(FLAGS[index])}
+      </option>
     );
 
     return (
       <div className={filtersPanel.filtersPanel}>
-        <select id="ISOfilter" onChange={(e) => this.props.onChange('flag', e.target.value)} defaultValue={0}>
-          <option value="" key={0}>Vessel flag (ISO code)</option>
+        <select id="ISOfilter" onChange={(e) => this.props.onChange('flag', e.target.value)} defaultValue={-1}>
+          <option value="" key={-1}>Vessel flag (ISO code)</option>
```

Comment:
I think you need to set the `value` field to `-1` so the `defaultValue` attribute on `select` is taken into account.